pkg/network/network.go

```diff
@@ -59,7 +73,11 @@ func (s *SkupperStatus) GetRouterSiteMap() map[string]SiteStatusInfo {
 			for _, routerStatus := range siteStatus.RouterStatus {
 				// the name of the router has a "0/" as a prefix that it is needed to remove
 				routerName := strings.Split(routerStatus.Router.Name, "/")
-				mapRouterSite[routerName[1]] = siteStatus
+
+				// Remove routers that belong to statefulsets for headless services
+				if strings.HasPrefix(routerName[1], siteStatus.Site.Namespace) {
```

Comment:
same here...

Reply:
Fixed in f2f23f1b0cde2c96ab96ffc027480a1c05f1899b